osm_validator_front/js/config/routing.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { BrowserRouter, Route, Redirect, Switch } from 'react-router-dom'
+import MainPage from '../components/mainpage.jsx'
+import Settings from '../components/settings.jsx'
+
+class Routing extends React.Component {
+    render() {
+        return (
+            <BrowserRouter>
```

Comment:
Lets use `HashRouter` instead, `BrowserRouter` require server support for page reloading